cornac/data/text.py

```diff
@@ -974,6 +974,7 @@ def batch_tfidf(self, batch_ids, keep_sparse=False):
         tfidf_mat = self.tfidf_matrix[batch_ids]
         return tfidf_mat if keep_sparse else tfidf_mat.A
```

Comment:
looks like we need to change this

Reply:
Yeah I missed it. Also modifying the same modification in same file, l.954 on bow_mat.